### PR TITLE
[STLND-702] Implement OnSent confirmation callback

### PR DESCRIPTION
**Pull request by jaszczur171**
- Added OnSent confirmation callback to the SDK's SendMessage method
- Introduced 'StellaNowSDK.Types' namespace for defining types
- Expanded SDK test suite with a test verifying the callback functionality
- Updated demo application to use the new OnSent callback feature

### CODE REVIEW

**Review by yan-vostrikov — DISMISSED** [review]
LGTM

**Review by badsector3 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -35,95 +36,115 @@ You will need to provide necessary credentials to interact with the Stella Now p
...
-            }
-    
-            // Disconnect and terminate
-            await stellaSdk.StopAsync();
-            DisposeServices();
-        }
-    
-        private static void RegisterServices()
+        // Establish connection
+        await _stellaSdk.StartAsync();
```
Comment (sebtom) [inline README.md:58]: adding ful blocks of code into documentation files might easily get outdated, or miss bug fix if we only change the sources

you can change this to reference code bocks directly from sources: https://www.stevemar.net/github-code-in-readme/

Comment (jaszczur171) [reply README.md:58]: This documentation is also uploaded as part of the NuGet deployment. The reference blocks do not work there as far as I am concerned.

Comment (sebtom) [reply README.md:58]: crap, have you tried or there is some article explaining limitation? maintaining both places manually is error-prone

**Pull request merged**